test/testdata/runner/two_questions.test.json

```diff
@@ -231,10 +217,24 @@
                         ],
                         "uuid": "ba96bf7f-bc2a-4873-a7c7-254d1927c4e3"
                     },
-                    "created_on": "2018-07-06T12:30:15.123456789Z",
+                    "created_on": "2018-07-06T12:30:11.123456789Z",
                     "step_uuid": "8720f157-ca1c-432f-9c0b-2014ddc77094",
                     "type": "contact_refreshed"
                 },
+                {
+                    "created_on": "2018-07-06T12:30:16.123456789Z",
+                    "msg": {
+                        "channel": {
+                            "name": "Nexmo",
+                            "uuid": "57f1078f-88aa-46f4-a59a-948a5739c03d"
+                        },
+                        "text": "I like blue!",
+                        "urn": "tel:+12065551212",
+                        "uuid": "9bf91c2b-ce58-4cef-aacc-281e03f69ab5"
+                    },
+                    "step_uuid": "8720f157-ca1c-432f-9c0b-2014ddc77094",
+                    "type": "msg_received"
+                },
```

Comment:
this is just an event ordering change - the `contact_refreshed` coming before the `msg_received`  so changes to last_seen_on aren't immediately lost by replacing the contact